test/remote/gateways/remote_wompi_test.rb

```diff
@@ -74,13 +74,13 @@ def test_successful_refund
     assert_success refund
   end
 
-  # def test_partial_refund
-  #   purchase = @gateway.purchase(@amount, @credit_card, @options)
-  #   assert_success purchase
+  def test_partial_refund
+    purchase = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success purchase
 
-  #   assert refund = @gateway.refund(@amount - 1, purchase.authorization)
-  #   assert_success refund
-  # end
+    assert refund = @gateway.refund(@amount - 50000, purchase.authorization)
```

Comment:
I `pry`ed into this test as well to check the @amount, also not seeing a change. I guess if you wanted to be more explicit with the test, you could add a couple of lines like
`response_data = refund.params['data']`
    `assert_equal response_data.dig('transaction', 'amount_in_cents'), 100000`
But as the gateway is currently behaving that assert_equal will fail.

Reply:
Thanks @rachelkirk ! I also experienced some intermittent failure on tests that seemed related to connection issues with the sandbox.

I will definitely wait on Wompi's feedback before merging, and if they indicate that we should be seeing the partial amount in their response, I'll dig into that further and also update the test with the `assert_equal` tip you suggested